### PR TITLE
Authentication controller

### PR DESCRIPTION
**Pull request by AndreiCinc**

### CODE REVIEW

File: reminderapp-be/src/main/java/event/ReminderApp/api/AuthenticationController.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    @Autowired
+    public AuthenticationController(AuthenticationService authenticationService) {
+        this.authenticationService = authenticationService;
+    }
+
+    @CrossOrigin
+    @PostMapping("authenticate/")
+    public String authenticate(@RequestBody User user) {
+        try{
```
Comment (andreisamoila74) [inline reminderapp-be/src/main/java/event/ReminderApp/api/AuthenticationController.java:23]:
```suggestion
        try {
```

File: reminderapp-be/src/main/java/event/ReminderApp/api/AuthenticationController.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            return authenticationService.Authenticate(user);
+        }catch (Exception e) {
```
Comment (andreisamoila74) [inline reminderapp-be/src/main/java/event/ReminderApp/api/AuthenticationController.java:25]:
```suggestion
        } catch (Exception e) {
```

**Review by andreisamoila74 — CHANGES_REQUESTED** [review]

File: reminderapp-be/src/main/java/event/ReminderApp/api/AuthenticationController.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+           throw new ApiRequestException("Failed to authenticate" + e.getMessage(), e.getCause());
```
Comment (andreisamoila74) [inline reminderapp-be/src/main/java/event/ReminderApp/api/AuthenticationController.java:26]: this line should be indented same as line 24

File: reminderapp-be/src/main/java/event/ReminderApp/api/AuthenticationController.java (hunk 1 above)
Comment (AndreiCinc) [reply reminderapp-be/src/main/java/event/ReminderApp/api/AuthenticationController.java]: Done

File: reminderapp-be/src/main/java/event/ReminderApp/api/AuthenticationController.java (hunk 2 above)
Comment (AndreiCinc) [reply reminderapp-be/src/main/java/event/ReminderApp/api/AuthenticationController.java]: done

File: reminderapp-be/src/main/java/event/ReminderApp/api/AuthenticationController.java (hunk 3 above)
Comment (AndreiCinc) [reply reminderapp-be/src/main/java/event/ReminderApp/api/AuthenticationController.java]: Done

**Pull request merged**